apps/site/next-data/providers/blogData.ts

```diff
@@ -1,10 +1,14 @@
 import { cache } from 'react';
 
-import generateBlogData from '@/next-data/generators/blogData.mjs';
 import { BLOG_POSTS_PER_PAGE } from '@/next.constants.mjs';
+import { blogData } from '@/next.json.mjs';
 import type { BlogCategory, BlogPostsRSC } from '@/types';
 
-const { categories, posts } = await generateBlogData();
+const { categories } = blogData;
```

Comment:
Feels weird to me a top level destructuring and then assigning posts on top level just for it to only be used on provideBlogPosts

Reply:
I think the destructuring is not needed, simply call `blogData.categories` on the cache call; And surroundd the post map to a function, even if it is top level, I don't want a `.map/.filter.etc` call on top level of the document; And no no need to move it to provideBlogPosts, only if you feel this context could change/invalidate between calls, which I doubt. Honestly speaking, even categories doesn't need to be behind `cache()`, since there's no operation there.

Reply:
Ok thanks for the clarification 🙂 

Is this ok? https://github.com/nodejs/nodejs.org/commit/56246dac394b3cb8fd928eb20e208329ffb4cd45

(PS: do let me know if you'd like me to remove the `cache()` call wrapping the categories)

Reply:
Updated 🙂 👍